### PR TITLE
Feature/content type actions

### PR DESCRIPTION
**Pull request by simonmudd**

### CODE REVIEW

File: src/lib/model/ContentType.ts [hunk 1]
```diff
@@ -95,7 +96,23 @@ export class ContentType extends HalResource {
   /**
    * Resources and actions related to a ContentType
    */
-  public readonly related = {};
+  public readonly related = {
```
Comment (easen-amp) [inline src/lib/model/ContentType.ts]: Missing some comments - they are used in the docs `npm run doc`

File: src/lib/model/ContentTypeSchema.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { HalResource } from '../hal/models/HalResource';
+import { Page } from './Page';
+
+export interface ContentTypeCachedSchema {
```
Comment (easen-amp) [inline src/lib/model/ContentTypeSchema.ts]: Missing comment block

File: src/lib/model/ContentType.ts (hunk 1 above)
Comment (simonmudd) [reply src/lib/model/ContentType.ts]: added in some supporting comments

File: src/lib/model/ContentTypeSchema.ts (hunk 2 above)
Comment (simonmudd) [reply src/lib/model/ContentTypeSchema.ts]: added in some supporting comments

Comment (easen-amp) [reply src/lib/model/ContentTypeSchema.ts]: Missing comment block for the `ContentTypeCachedSchema` interface too

**Pull request merged**